### PR TITLE
Define global object's CookieStore more formally

### PR DESCRIPTION
**Pull request by ayuishii**
Discussion: https://github.com/WICG/cookie-store/issues/158

***
<a href="https://pr-preview.s3.amazonaws.com/ayuishii/cookie-store/pull/172.html" title="Last updated on Sep 10, 2020, 5:29 PM UTC (b5c3736)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/WICG/cookie-store/172/1d2a91f...ayuishii:b5c3736.html" title="Last updated on Sep 10, 2020, 5:29 PM UTC (b5c3736)">Diff</a>

### CODE REVIEW

File: index.bs [hunk 1]
```diff
@@ -1019,7 +1019,9 @@ partial interface Window {
 };
 </xmp>
 
-The <dfn attribute for=Window>cookieStore</dfn> getter steps are to return [=/this=]'s [=relevant settings object=]'s {{CookieStore}} object.
+A {{Window}} has an <dfn attribute for=Window>associated CookieStore</dfn>, which is a {{CookieStore}}.
```
Comment (inexorabletash) [inline index.bs:1022]: I don't know that we want `attribute` here - are you following precedent somewhere?

File: index.bs [hunk 2]
```diff
@@ -1019,7 +1019,9 @@ partial interface Window {
...
+
+The <dfn attribute for=Window>cookieStore</dfn> getter steps are to return [=/this=]'s {{Window/associated CookieStore}}.
```
Comment (inexorabletash) [inline index.bs:1024]: And we definitely don't want this to be an IDL link `{{ ... }}`, I think just a regular `[=Window/associated CookieStore=]` link ?

File: index.bs [hunk 3]
```diff
@@ -1033,7 +1035,9 @@ partial interface ServiceWorkerGlobalScope {
 };
 </xmp>
 
-The <dfn attribute for=ServiceWorkerGlobalScope>cookieStore</dfn> getter steps are to return [=/this=]'s [=relevant settings object=]'s {{CookieStore}} object.
+A {{ServiceWorkerGlobalScope}} has an <dfn attribute for=ServiceWorkerGlobalScope>associated CookieStore</dfn>, which is a {{CookieStore}}.
```
Comment (inexorabletash) [inline index.bs:1038]: ditto

File: index.bs [hunk 4]
```diff
@@ -1033,7 +1035,9 @@ partial interface ServiceWorkerGlobalScope {
...
+
+The <dfn attribute for=ServiceWorkerGlobalScope>cookieStore</dfn> getter steps are to return [=/this=]'s {{ServiceWorkerGlobalScope/associated CookieStore}}.
```
Comment (inexorabletash) [inline index.bs:1040]: ditto

**Review by inexorabletash — CHANGES_REQUESTED** [review]

File: index.bs (hunk 1 above)
Comment (ayuishii) [reply index.bs]: Ah sorry, just a combination of not knowing what I'm doing and trying to make the links work 🤦‍♀️ .
Thanks for the help!

**Review by inexorabletash — APPROVED** [review]

**Pull request merged**